### DISCUSSION TITLE
Phone Number Input

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2483] [Q&A] [answered]
Hi team, I'm working on migrating the phone number verification page to Paste and I accidentally dropped in the phone number input [component from Match](https://www.figma.com/file/oYazzpZ4XWFUlwD1pDHNms/Core-Components?node-id=1161%3A2738)--oops! I saw a discussion on this board from Ishita from two years ago, I'm wondering if there's been any further talk or requests from teams to have this. 

(Edit from @serifluous: Here's the link to the related discussion #695 )

**Comment by serifluous** [comment, marked as answer]
Hey @rosemaryluo ! Our answer unfortunately hasn't changed much from [Ishita's original discussion](https://github.com/twilio-labs/paste/discussions/695).

Handling phone number validation requires more business logic, understanding of how our customers interact with these numbers, and a solid understanding of the differences across countries that the Phone Numbers team would be much better equipped to handle. I think your best bet here would be to collaborate with the designers on that team on best practices in handling phone numbers, and seeing if their team has plans to build a shared component for this use case.

Please let us know if you do this! We'd love to publish some guidelines around how to handle phone number validation, but the Numbers team would be best equipped to build it on top of the Paste Input component and maintain it.
